rbac/server/api/users.py

```diff
@@ -89,6 +90,9 @@
     encrypted_private_key = encrypt_private_key(
         AES_KEY, key_pair.public_key, key_pair.private_key_bytes
     )
+    metadata1 = request.json.get("metadata")
```

Comment:
I just noticed this was our create_user API endpoint. We would only want to create users within NEXT only and not have to send it out to our AD or Azure provider. So this API endpoint should not need a flag.